File: app/src/main/java/co/tpcreative/supersafe/model/GalleryAlbum.java
package co.tpcreative.supersafe.model;

import co.tpcreative.supersafe.common.entities.MainCategoryEntity;

public class GalleryAlbum {
    public final int videos;
    public final int photos;
    public final int audios;
    public final int others;
    public final MainCategoryModel main;


    public GalleryAlbum(MainCategoryModel main, int photos, int videos, int audios, int others){
        this.main = main;
        this.photos = photos;
        this.videos = videos;
        this.audios = audios;
        this.others = others;
    }
}

File: app/src/main/java/co/tpcreative/supersafe/model/GalleryAlbum.kt
package co.tpcreative.supersafe.model

class GalleryAlbum(main: MainCategoryModel?, photos: Int, videos: Int, audios: Int, others: Int) {
    val videos: Int
    val photos: Int
    val audios: Int
    val others: Int
    val main: MainCategoryModel?

    init {
        this.main = main
        this.photos = photos
        this.videos = videos
        this.audios = audios
        this.others = others
    }
}